scripts/chocolatey-packages.bat

```diff
@@ -13,4 +15,5 @@ set choco_binary=c:\programdata\chocolatey\bin\choco.exe
 %choco_binary% install microsoft-build-tools -y
 %choco_binary% install visualcppbuildtools -y
 setx /M VCTargetsPath "C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\v140"
-
+%choco_binary% Install-ChocolateyEnvironmentVariable "JAVA_HOME" "C:\Program Files\Java\jdk1.8.0_211"
```

Comment:
The build of the images is performed every two months automatically, what happen with this set of environment variables if the version of jdk or maven changes in the chocolatey repositories?